### PR TITLE
Adds a pipeline example for compressing a file in Cloud Storage

### PR DESCRIPTION
**Pull request by mbookman**
Also provides an example of an ephemeral pipeline.

### CODE REVIEW

File: compress/test/cloud/test_run_gzip.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      'disks': [ {
+        'name': 'data',
+        'autoDelete': True,
+
+        # Within the docker container, specify a mount point for the disk.
+        # The pipeline input argument below will specify that inputs should be
+        # written to this disk.
+        'mountPoint': '/mnt/data',
+
+        # Specify a default size and type
```
Comment (deflaux) [inline compress/test/cloud/test_run_gzip.py]: nit: some comments do not end with a period.

**Comment by deflaux** [comment]
@mbookman Great sample - nicely descriptive comments!

Consider renaming test_run_gzip.py to run_gzip.py.  LGTM

Note that the `test` subdirectory seems unnecessary in this example and in the samtools example.  Subdirectories `local` and `cloud` seem sufficient or `local_test` and `cloud`.

File: compress/test/cloud/test_run_gzip.py (hunk 1 above)
Comment (mbookman) [inline compress/test/cloud/test_run_gzip.py]: Thanks- fixed them to be consistent with single-sentence comments omitting periods and mutli-sentence comments using periods.

File: compress/test/cloud/test_run_gzip.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python sample demonstrating use of the Google Genomics Pipelines API.
+
+This sample demonstrates running a pipeline to compress a file that is in
+Google Cloud Storage.
+
+This sample demonstrates running the pipeline in an "ephemeral" manner;
+no call to pipelines.create() is neccessary. No pipeline is persisted
```
Comment (pgrosu) [inline compress/test/cloud/test_run_gzip.py]: s/neccessary/necessary

**Pull request merged**